### ISSUE TITLE
Dispose of unclaimed balances

### ISSUE THREAD

**Issue by chadwhitacre** [gratipay/inside.gratipay.com#1205]
Reticketing from #1196.

- [x] high-touch for fat head: #1225
- [x] masspay for long tail: #1219
- [x] masspay for the longer tail: #1226 &rarr; #1219
- [x] wait 30 days for PayPal MassPay dunning
- [x] meanwhile, dun non-MassPay pending payouts manually
- [x] when dunning is over, prep another MassPay for failed accounts for which we have additional email addresses
- [x] ~~maybe wait 30 days and do a third MassPay? Need to look at amounts ...~~ probably not going to be worth it: https://github.com/gratipay/inside.gratipay.com/issues/1205#issuecomment-449612538
- [x] donate the rest

**Comment by chadwhitacre** [comment]
http://www.patreasury.gov/unclaimed-property/

http://patreasury.gov/unclaimed-property/holder/

http://patreasury.gov/pdf/unclaimed-property/Voluntary-Disclosure-Agreement.pdf

**Comment by chadwhitacre** [comment]
http://www.unclaimedpropertyspecialists.com/contact

**Comment by chadwhitacre** [comment]
> Greetings! I have been running a third-party payment processor for the past five years and we are now shutting down our service. We are sitting on approximately $100,000 of escrowed funds and are looking for advice on disposing of it.

**Comment by chadwhitacre** [comment]
Had a call with MarketSphere. They specialize in handling filings for unclaimed property but are not lawyers. The person I talked with was going to recommend some lawyers who could go over our [terms](https://gratipay.com/about/policies/terms-of-service) and advise us on our liability.

From an unclaimed property point of view, it's unfortunate that don't have national identification info for everyone we're holding money for. At this point in a "clean environment" (i.e., w/ identification on file) we would send hard-copy paper letters to the last known address for everyone affected for three years before turning over the list of names/addresses and dollar amounts to the state of the last known address. For international or unknown people the property reverts to our state of incorporation, namely the Commonwealth of Pennsylvania. The fact that we don't have national identification information on file is a red flag, and could potentially lead to an "[unclaimed property audit](https://www.natlawreview.com/article/unclaimed-property-audits-no-laughing-matter)." Pennsylvania would be the most likely to be interested in conducting an audit, though the person I spoke with considered our exposure to be "minimal" and an audit to be "extremely unlikely." They went so far as to suggest, after I mentioned [Readability](https://github.com/gratipay/gratipay.com/issues/3539#issuecomment-122927819) (btw, archive: [.org](https://web.archive.org/web/20150730191851/http://blog.readability.com/2012/06/announcement/), [.png](https://user-images.githubusercontent.com/134455/50380932-ca1ebc00-0646-11e9-944d-9da92679048b.png)), that donating the money in a well-documented fashion would be an option worth considering. (Again, not a lawyer.) However, donating the money would create liability for whomever we were donating it to: we'd have to be up-front with them about the source of the funds and make sure they were okay with it. I think both the letter and the spirit of the law suggest that we give the money to the Commonwealth of Pennsylvania.

So basically what we'll do is make a one-time [filing](http://patreasury.gov/unclaimed-property/holder/) with the Commonwealth. The general expectation is that companies who file, file year after year, so we need to make clear that we're closing up shop and this will be our one and only filing, relinquishing all of our liability, and that there won't be any future filings. We'll give them what we have: usernames, email addresses, social media accounts, dollar amounts, and transaction dates. We'll send the money via wire/ACH/check. We'll include a narrative describing the source of the funds and our attempts to notify owners and return the money. Even though the dormancy period is not up we can "force file" because we are going out of business.

Hopefully we can get this done in Q1 2018.

**Comment by chadwhitacre** [comment]
I've received from MarketSphere a list of law firms that specialize in unclaimed property.

**Comment by chadwhitacre** [comment]
Where we have an email address, we can try sending via PayPal even if not explicitly linked to PayPal.

```
gratipay-bak=# select count(*), sum(balance) volume, email_address is not null as have_email_address from participants where balance > 0 group by email_address is not null;
┌───────┬──────────┬────────────────────┐
│ count │  volume  │ have_email_address │
├───────┼──────────┼────────────────────┤
│  3162 │ 91861.94 │ f                  │
│   602 │ 12928.67 │ t                  │
└───────┴──────────┴────────────────────┘
(2 rows)

gratipay-bak=#
```

**Comment by chadwhitacre** [comment]
Looks like that's only about 10%. :-/

**Comment by chadwhitacre** [comment]
@JessWhit suggests that we should block this on https://github.com/gratipay/inside.gratipay.com/issues/1220.

**Comment by chadwhitacre** [comment]
> This [evening](https://www.meetup.com/Pittsburgh-Code-Supply/events/256571013/), I hand-delivered $420 in cash to someone who is friends with someone who had money escrowed with us. 💃

https://github.com/gratipay/inside.gratipay.com/issues/1219#issuecomment-442311277

**Comment by chadwhitacre** [comment]
I reimbursed myself the $420 via a cash withdrawal from New Alliance.

**Comment by chadwhitacre** [comment]
I note that the tax situation for recipients is as muddy as ever. We are the payment processor, not the employer/contracting company.

Is it better to push the money back upstream? 🤔

**Comment by chadwhitacre** [comment]
Because technically the money was income to people the moment they received it through Gittip/Gratipay, whether or not they reported it. Now it's just a pain. 😞

Would it be less inconvenient for the people who gave it originally to receive it back?

I am revisiting the idea of donating it to open source non-profits.

**Comment by chadwhitacre** [comment]
I sent money to one account under #1225 and emailed the other 12.

**Comment by chadwhitacre** [comment]
By quartile:

```
   n  cutoff
------------
3466   69.00
 217  227.00
  67  838.50
  13 4306.93
```

**Comment by chadwhitacre** [comment]
```
gratipay-bak=# \i charts.sql
ALTER TABLE
UPDATE 3467
UPDATE 217
UPDATE 67
UPDATE 13
ALTER TABLE
UPDATE 2112
┌──────┬──────────┐
│  n   │    s     │
├──────┼──────────┤
│   13 │ 24934.11 │
│   67 │ 27197.25 │
│  217 │ 26367.88 │
│ 3467 │ 26291.37 │
└──────┴──────────┘
(4 rows)

┌──────┬──────────┐
│  n   │    s     │
├──────┼──────────┤
│    7 │ 14153.18 │
│   42 │ 16848.07 │
│  125 │ 15790.67 │
│ 1938 │ 15280.23 │
└──────┴──────────┘
(4 rows)

gratipay-bak=#
```

```sql
alter table tmp add column bucket int default 0;
update tmp set bucket=0 where balance > 0 and balance <= 69;
update tmp set bucket=1 where balance > 69 and balance <= 227;
update tmp set bucket=2 where balance > 227 and balance <= 838.50;
update tmp set bucket=4 where balance > 838.50;

alter table tmp add column has_email bool default false;
update tmp
   set has_email=true
 where coalesce(address_1, address_2, address_3, address_4, address_5) is not null
      ;

select count(balance) n, sum(balance) s from tmp group by bucket order by bucket desc;
select count(balance) n, sum(balance) s from tmp where has_email group by bucket order by bucket desc;
```

```python
#!/usr/bin/env python3
import csv
from collections import OrderedDict
from decimal import Decimal as D

payouts = csv.reader(open('payouts.csv'))
masspay = csv.writer(open('masspay.csv', 'w+'))
headers = next(payouts)
blacklist = set(open('blacklist.csv').read().splitlines())
TOTAL = total = settled = D('0.00')
amounts = []

for row in payouts:
    paid, balance = bool(int(row[0])), D(row[1])
    amounts.append(balance)
    TOTAL += balance
    if paid:
        settled += balance
        continue
    addresses = list(OrderedDict.fromkeys(row[4:]))
    for address in addresses:
        if not address or address in blacklist:
            continue
        masspay.writerow([address, balance, 'usd'])
        total += balance
        break

print(f'{settled} / {total} ({(settled / total * 100):.1f}%)')

# Print out quartiles
quartile = TOTAL / 4
current = quartile
print(TOTAL, quartile)
running = D('0.00')
amounts.sort()
j = 0

def log():
    print(f"{current:>9.02f} {running:>9.02f} {i-j:>4d} {amount:>7.02f}")

for i, amount in enumerate(amounts):
    if running >= current:
        log()
        current += quartile
        j = i
    running += amount
log()

```

**Comment by chadwhitacre** [comment]
Based on https://github.com/gratipay/inside.gratipay.com/issues/1205#issuecomment-446011658 I think we should manually reach out (#1225) to the top two quartiles. Sending another 67 emails in order to clear another 25% of the escrow is a good trade-off. From there we can move back to #1219 for the bottom half.

**Comment by chadwhitacre** [comment]
It seems like I should spin Gratipay.com back up locally and track these exchanges so we have a clear record.

**Comment by chadwhitacre** [comment]
```
gratipay-bak=# select done, sum(balance) from tmp group by done;
┌──────┬──────────┐
│ done │   sum    │
├──────┼──────────┤
│ f    │ 45282.42 │
│ t    │ 59508.19 │
└──────┴──────────┘
(2 rows)
```

**Comment by chadwhitacre** [comment]
Alright! MassPay Number 1 is done! Well, started. It looks like we successfully sent $31k+ (30%) right off the bat, with another $11k (10%) still potentially to be claimed.

We're now down to 1700 that aren't done and have no email. Here's the breakdown:

```
gratipay-bak=# select bucket, count(balance), sum(balance) from tmp where not done and n = 0 group by bucket order by bucket;
┌────────┬───────┬──────────┐
│ bucket │ count │   sum    │
├────────┼───────┼──────────┤
│      1 │  1580 │ 11385.23 │
│      2 │    95 │ 10961.71 │
│      3 │    25 │  9732.61 │
└────────┴───────┴──────────┘
(3 rows)
```

The buckets are:

1. 0 — 69
1. 69 — 227
1. 227 — 834

I think I'll take a pass through buckets 2 and 3 and clear low-hanging fruit.

**Comment by chadwhitacre** [comment]
Note that the "done" amount of $59k includes $9k in pending payouts to the five top-13 folks that are actually still pending (#1225) though I've made initial contact with four and have a plan for the fifth.

**Comment by chadwhitacre** [comment]
The no email numbers are a little fudgy, another way I counted them I got 1650. ¯\\\_(ツ)\_/¯

| category | amount ($) | % | estimate | $ |
|:------|--------:|------:|-----:|------:|
| total | 104,790.61 | | | |
| done done! | 50,357.64 | 48.1 | |
| remaining | 54,432.97 | 51.9 | | |
| pending on #1225 | 9,150.55 | 8.7 | 1.0 | 9,150.55 |
| pending on #1219 | 12,302.29 | 11.7 | 0.5 | 6,151.15 |
| denied on #1219 | 2,318.57 | 2.2 | 0.0 | 0 |
| no email | 30,661.22 | 29.3 | 0.1 | 3,066.12 |
| remaining maybe? | | | | 18,367.82 |
| total maybe? | | 65.6 | | 68,725.46 |

**Comment by chadwhitacre** [comment]
> another way

https://github.com/gratipay/logs/commit/c69e005691a5c741590d75e81586d2aff667c947

**Comment by chadwhitacre** [comment]
^^^ Table updated with some estimates of what we'll be able to flush. Target: 2/3! 👍

**Comment by chadwhitacre** [comment]
I think I can surpass $3,066.12 for `no email` (`state=4`):

```
gratipay-bak=# select count(balance), sum(balance) from tmp where state=4 and balance >= 100;
┌───────┬──────────┐
│ count │   sum    │
├───────┼──────────┤
│    75 │ 16225.47 │
└───────┴──────────┘
(1 row)

gratipay-bak=# 
```

Excelsior!

**Comment by chadwhitacre** [comment]
Per the above table there would be $36,065.15 remaining.

2/3 gone would be $34,930.20 left.

Anything under $40,000 and especially if we could get under $30,000(!) I would start to feel pretty good about donating the rest to selected charities.

**Comment by chadwhitacre** [comment]
lol that's one way

![screen shot 2018-12-17 at 6 52 46 am](https://user-images.githubusercontent.com/134455/50085582-76385100-01c8-11e9-9b26-f536e6e3039b.png)

**Comment by chadwhitacre** [comment]
lol owed Hank Green dollars:

![screen shot 2018-12-17 at 10 50 31 am](https://user-images.githubusercontent.com/134455/50098291-9f1d0e00-01e9-11e9-8d56-ab5a22917e9e.png)

**Comment by chadwhitacre** [comment]
double lol

![screen shot 2018-12-17 at 10 52 38 am](https://user-images.githubusercontent.com/134455/50098480-05099580-01ea-11e9-9baf-8d17faa96c7e.png)

**Comment by chadwhitacre** [comment]
I requested another PayPal Debit Card so I can pay out some accounts where there's a donation option on their website but they don't support PayPal.

![screen shot 2018-12-17 at 11 02 07 am](https://user-images.githubusercontent.com/134455/50099101-43ec1b00-01eb-11e9-8428-9786242288a7.png)

**Comment by chadwhitacre** [comment]
tfw you manually trigger GitHub's rate-limiting 😬

![screen shot 2018-12-17 at 11 15 45 am](https://user-images.githubusercontent.com/134455/50099881-24ee8880-01ed-11e9-8b43-55317e63f713.png)

**Comment by chadwhitacre** [comment]
$20,000 has cleared New Alliance.

PayPal debit card is in the mail.

I am updating the script to process payments.

Then:

_[folded into ticket description]_

**Comment by chadwhitacre** [comment]
Email harvesting script moved to a Gist:

https://gist.github.com/chadwhitacre/d4cb21261e41a2da1dc3bc7f86e6879c

**Comment by chadwhitacre** [comment]
Scripts updated in https://github.com/gratipay/logs/commit/c8d654b7157aed9194beb982d07ad2c05535c0b4.

```
$ ./tick.py 
               n                $
           ------------- ------------------
    ready     96 ( 2.6%)  15,280.99 (14.6%)
  pending    512 (13.6%)  14,473.53 (13.8%)
 complete  1,538 (40.9%)  61,690.20 (58.9%)
   failed  1,618 (43.0%)  13,345.89 (12.7%)
    total  3,764         104,790.61

  masspay     96 ( 2.6%)
   target                 15,280.99
remainder               -      0.00
                         ----------
   actual                 15,280.99 (14.6%)
      fee               -     91.28
                         ----------
      net                 15,189.71
$
```

**Comment by chadwhitacre** [comment]
Okay! Gonna take five, and then see about issuing this next masspay ...

**Comment by chadwhitacre** [comment]
Current balances:

| Account | Amount ($) |
|:-------|---------:|
| New Alliance | 19,684.45 |
| PayPal | 3,243.00 |
| total | 22,927.45 | 

The PayPal balance doesn't include pending money, so that's going to spring back significantly.

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/7ed351d3c8e3edaa005104a1774b9d927c192b23

```
$ ./tick.py 
               n                $
           ------------- ------------------
    ready      0 ( 0.0%)       0.00 ( 0.0%)
  pending    475 (12.6%)  14,617.08 (13.9%)
 complete  1,663 (44.2%)  76,175.55 (72.7%)
   failed  1,626 (43.2%)  13,997.98 (13.4%)
    total  3,764         104,790.61

  masspay      0 ( 0.0%)
   target                      0.00
remainder               -      0.00
                         ----------
   actual                      0.00 ( 0.0%)
      fee               -      0.00
                         ----------
      net                      0.00
$
```

**Comment by chadwhitacre** [comment]
Damn. This feels like we are succeeding here!

Emails deduped in `payouts.csv` in https://github.com/gratipay/logs/commit/9ae34407f88f8d52bfd9413278cfa3f8dd14c355.

```python
#!/usr/bin/env python
import csv
import lib


payouts_header, payouts = lib.load_payouts()
for row in payouts:
    addresses = lib.get_addresses(row)
    addresses += ([''] * (4-len(addresses)))
    row[5:] = addresses

csv.writer(open('payouts.csv', 'w+')).writerows([payouts_header] + payouts)
```

None of the current failures have a second email address. So they are failed indeed.

```
$ grep ',unaddressable,failed.*@.*@' payouts.csv
$
```

... or are they!?

I've gone through five failed payouts > $100 (not including ones where I couldn't find an email address the [other day](https://github.com/gratipay/inside.gratipay.com/issues/1219#issuecomment-447988711)), one I was able to pay out via PayPal based on an updated email address from GitHub, the other four I emailed to ask for options. One of them is @techtonik! 👋 😍

Forty of the pending masspays have a second email address, so I guess that's an upper limit on the third MassPay we're expecting to do in ~30 days when the results are in from the first two. With those kind of numbers I don't anticipate a fourth.

```
$ grep 'masspay,pending.*@.*@' payouts.csv | wc -l
      40
$
```

**Comment by chadwhitacre** [comment]
Got the debit card!

Five people to pay ...

**Comment by chadwhitacre** [comment]
First wanted Cash.me. Won't take the new debit card because "blah blah issuing bank charges additional fees" 😖. Tried my own card "that looks like a credit card" tried my own other card. Worked! PayPaling myself the reimbursement ...

**Comment by chadwhitacre** [comment]
The rest of the debit card folks were straightforward. 👍

https://github.com/gratipay/logs/commit/0458e7288a60dc10ce4830193f019c385cc9c2e7

```
$ ./tick.py 
               n                $
           ------------- ------------------
    ready      0 ( 0.0%)       0.00 ( 0.0%)
  pending    473 (12.6%)  14,969.60 (14.3%)
 complete  1,670 (44.4%)  77,566.64 (74.0%)
   failed  1,621 (43.1%)  12,254.37 (11.7%)
    total  3,764         104,790.61

  masspay      0 ( 0.0%)
   target                      0.00
remainder               -      0.00
                         ----------
   actual                      0.00 ( 0.0%)
      fee               -      0.00
                         ----------
      net                      0.00
$
```

**Comment by chadwhitacre** [comment]
Yeah, this is going great. We've got ~$1,600 in flight on TransferWise that is all but certain to clear on Monday, plus five Patreon payouts that are supposed to clear on Jan 1. Those alone will put us over 3/4 complete at $79,867.06 (76.2%). 👍

Waddya think? Can we make 80%? 💃

**Comment by chadwhitacre** [comment]
Can we semi-automate harvesting GitHub emails for the 1500+ people that are failed unaddressable?

**Comment by chadwhitacre** [comment]
Because I'm pretty sure we were just looking at emails explicitly published on a GitHub account, vs. emails on commit logs.

**Comment by chadwhitacre** [comment]
That sounds like a new ticket: https://github.com/gratipay/inside.gratipay.com/issues/1226.

**Comment by chadwhitacre** [comment]
Gosh. What if we could clear 90%? I think we might be able to get there with #1226, though that will take a lot of manual review. It would really be worth it, though. If we have less than $10,000 left unsendable then we could really declare victory! And then just donate the rest.

I went back and revisited the [Readability case](https://web.archive.org/web/20150730191851/http://blog.readability.com/2012/06/announcement/). They donated 90% of all of the money they collected, which came to about $150,000. We collected $1,003,648.55 over the lifetime of Gratipay:

```
gratipay-bak=# select sum(amount) from exchanges where amount > 0 and status = 'succeeded';
┌────────────┐
│    sum     │
├────────────┤
│ 1003648.55 │
└────────────┘
(1 row)

gratipay-bak=#
```

So to only have $104,790.61 left to dispose of is _already_ only 10.4% of our total volume—the inverse of Readability's situation. And as of today we've cleared $78,266.88 of that, leaving only $26,523.73, which is only 2.6% of our total volume ever processed. In other words, we're already in a very different situation than Readability, and I think we can feel good about where we stand today even if we make no further progress. Stretch goal: < 1% remaining! That'd be < $10,036.48. Can we get there?!?!? 😮

**Comment by chadwhitacre** [comment]
75% 💃

https://github.com/gratipay/logs/commit/05795f25caa75c3c28e9881250d031194bc47c54

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    471 (12.5%)  13,890.95 (13.3%)
   complete  1,672 (44.4%)  78,645.29 (75.0%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,092 (55.6%)  26,145.32 (25.0%)
$
```

**Comment by chadwhitacre** [comment]
75.3%

https://github.com/gratipay/logs/commit/c96285952cd6f5bcd7b0d00aa30f0d14b3086eec

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    469 (12.5%)  13,660.95 (13.0%)
   complete  1,674 (44.5%)  78,875.29 (75.3%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,090 (55.5%)  25,915.32 (24.7%)
$
```

**Comment by chadwhitacre** [comment]
I updated the blog post to emphasize the $1,000,000 base:

![screen shot 2018-12-24 at 2 52 40 pm](https://user-images.githubusercontent.com/134455/50406029-9c8f5b00-078b-11e9-88ff-6254bb22c159.png)

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/a41024a92ec651e6b514a267f2dbce4d570d689c

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    469 (12.5%)  13,660.95 (13.0%)
   complete  1,674 (44.5%)  78,875.29 (75.3%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,090 (55.5%)  25,915.32 (24.7%)

     volume              1,003,648.55
  disbursed                977,733.23 (97.4)%
$
```

**Comment by chadwhitacre** [comment]
When we started this would've been:

```
     volume              1,003,648.55
  disbursed                898,857.94 (89.6)%
```

**Comment by chadwhitacre** [comment]
#1225 is done! 💃

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    468 (12.4%)  12,062.28 (11.5%)
   complete  1,675 (44.5%)  80,473.96 (76.8%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,089 (55.5%)  24,316.65 (23.2%)

     volume              1,003,648.55
  disbursed                979,331.90 (97.6)%
$
```

**Comment by chadwhitacre** [comment]
🙈

> We’re also getting in touch to let you know that it’s going to take a bit longer than initially estimated to review your business.
> 
> TransferWise has an acceptable use policy ([see here](https://transferwise.com/acceptable-use-policy-eea)) and depending on the business type, we sometimes need to do a more in depth review to make sure that every business that uses TransferWise compliant with regulatory requirements.

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/6939b420caf138a81d5bc9a4f5a5626e7689b5ef

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    466 (12.4%)  11,937.31 (11.4%)
   complete  1,677 (44.6%)  80,598.93 (76.9%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,087 (55.4%)  24,191.68 (23.1%)

     volume              1,003,648.55
  disbursed                979,456.87 (97.6)%
$
```

**Comment by chadwhitacre** [comment]
Topped up the seven remaining pending on Twitter. I guess PayPal sends repeated dunning emails.

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/746b9b3c66084acfba1064a4624799007d478114

```
$ ./tick.py
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    461 (12.2%)  11,215.56 (10.7%)
   complete  1,682 (44.7%)  81,320.68 (77.6%)
     failed  1,621 (43.1%)  12,254.37 (11.7%)
      total  3,764         104,790.61
 incomplete  2,082 (55.3%)  23,469.93 (22.4%)

     volume              1,003,648.55
  disbursed                980,178.62 (97.7)%
$ 
```

**Comment by chadwhitacre** [comment]
Patreon hit, and four of the five pending there are now complete. One is per video so we have to wait for them to make a video.

```
$ grep pending payouts.csv | cut -d, -f2 | sort | uniq -c | sort -nr
 450 masspay
   7 twitter
   2 email
   1 patreon
   1 check
$
```

**Comment by chadwhitacre** [comment]
Heading back into #1219 after #1226. 👍

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready    350 ( 9.3%)   2,959.62 ( 2.8%)
    pending    462 (12.3%)  11,257.56 (10.7%)
   complete  1,683 (44.7%)  81,326.50 (77.6%)
     failed  1,269 (33.7%)   9,246.93 ( 8.8%)
      total  3,764         104,790.61
 incomplete  2,081 (55.3%)  23,464.11 (22.4%)

     volume              1,003,648.55
  disbursed                983,144.06 (98.0)%

    masspay    350 ( 9.3%)
     target                  2,959.62
  remainder               -      0.65
                           ----------
     actual                  2,958.97 ( 2.8%)
        fee               -     56.08
                           ----------
        net                  2,902.89
$
```

**Comment by chadwhitacre** [comment]
With https://github.com/gratipay/logs/commit/68afec6e8b937a8de68120729b102126ec1c9273 we are  up over 80% disbursed from the leftovers, and we are almost at 98% of total volume disbursed! 💃

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    578 (15.4%)  11,198.65 (10.7%)
   complete  1,904 (50.6%)  84,156.87 (80.3%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,860 (49.4%)  20,633.74 (19.7%)

     volume              1,003,648.55
  disbursed                983,014.81 (97.9)%
     target                983,575.58 (98.0)%
   distance                    560.77
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 566 masspay
   7 twitter
   3 email
   1 patreon
   1 check
$
```

**Comment by chadwhitacre** [comment]
Steady march! Eeeeeeee! 😍

https://github.com/gratipay/logs/commit/c7e55cf9343ebcd5a5d0906531bde20be100bae6

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    576 (15.3%)  11,165.53 (10.7%)
   complete  1,906 (50.6%)  84,189.99 (80.3%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,858 (49.4%)  20,600.62 (19.7%)

     volume              1,003,648.55
  disbursed                983,047.93 (97.9)%
     target                983,575.58 (98.0)%
   distance                    527.65
$
```

**Comment by chadwhitacre** [comment]
Yay rounding! 😬

https://github.com/gratipay/logs/commit/6f3dddae1d9c94d5ffabec054a650928789ebe47

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    573 (15.2%)  11,097.68 (10.6%)
   complete  1,909 (50.7%)  84,257.84 (80.4%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,855 (49.3%)  20,532.77 (19.6%)

     volume              1,003,648.55
  disbursed                983,115.78 (98.0)%
     target                983,575.58 (98.0)%
   distance                    459.80
$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/34bab4c091644d4176b3fa02c314d6c8b384c0f8

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    560 (14.9%)  10,833.98 (10.3%)
   complete  1,922 (51.1%)  84,521.54 (80.7%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,842 (48.9%)  20,269.07 (19.3%)

     volume              1,003,648.55
  disbursed                983,379.48 (98.0)%
     target                983,575.58 (98.0)%
   distance                    196.10
```

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 549 masspay
   7 twitter
   3 email
   1 patreon
$
```

**Comment by chadwhitacre** [comment]
I found a donation page for one of the larger pending MassPay balances! We have officially cleared 98%! ☺️

https://github.com/gratipay/logs/commit/2e63a94002474dbe1d418f0ef22f0d4a8656ee25

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    559 (14.9%)  10,385.90 ( 9.9%)
   complete  1,923 (51.1%)  84,969.62 (81.1%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,841 (48.9%)  19,820.99 (18.9%)

     volume              1,003,648.55
  disbursed                983,827.56 (98.0)%
     target                983,575.58 (98.0)%
   distance                   -251.98
$
```

![screen shot 2019-01-03 at 10 55 06 am](https://user-images.githubusercontent.com/134455/50647359-5c2b9c00-0f46-11e9-9200-f38f1aad05e4.png)

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 548 masspay
   7 twitter
   3 email
   1 patreon
$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/693ea38341c1b3a452bfd3e7dc6cbbf0200fa1df

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    554 (14.7%)   9,375.45 ( 8.9%)
   complete  1,928 (51.2%)  85,980.07 (82.0%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,836 (48.8%)  18,810.54 (18.0%)

     volume              1,003,648.55
  disbursed                984,838.01 (98.1)%
     target                983,575.58 (98.0)%
   distance                 -1,262.43
$
```

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 543 masspay
   6 twitter
   3 email
   1 patreon
   1 github
$
```

**Comment by chadwhitacre** [comment]
Another higher-value MassPay pending picked off in https://github.com/gratipay/logs/commit/7d425c1d4a238381ba1852d9193d8be31a0a51b4. 👍

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    553 (14.7%)   9,058.15 ( 8.6%)
   complete  1,929 (51.2%)  86,297.37 (82.4%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,835 (48.8%)  18,493.24 (17.6%)

     volume              1,003,648.55
  disbursed                985,155.31 (98.2)%
     target                983,575.58 (98.0)%
   distance                 -1,579.73
$ 
$ 
$ 
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 542 masspay
   6 twitter
   3 email
   1 patreon
   1 github
$
```

**Comment by chadwhitacre** [comment]
The boot from TransferWise! 👢

> Hi there Chad,
> 
> Thanks for your patience as we looked into the details of your business here. Much appreciated.
> 
> After review, it has been determined that we unfortunately aren’t able to assist with the activity you’ve outlined.
> 
> This is due to the nature of the funds you’d be dispersing through our system. As they’re being held in escrow, it would not be feasible for us to properly understand the source of the funds for the money you’d be moving.
> 
> I apologize for any inconvenience this may cause – I wish we were able to assist you further in this matter.
> 
> Kind regards,
> 
> []
> TransferWise

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/5de99dc258266d929970ff6fe314265d757bef17

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    551 (14.6%)   9,055.20 ( 8.6%)
   complete  1,931 (51.3%)  86,300.32 (82.4%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,833 (48.7%)  18,490.29 (17.6%)

     volume              1,003,648.55
  disbursed                985,158.26 (98.2)%
     target                983,575.58 (98.0)%
   distance                 -1,582.68
$
```

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 540 masspay
   6 twitter
   3 email
   1 patreon
   1 github
$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/c5b0206f8509458887d39c22711be893a08b27bd

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    547 (14.5%)   8,879.54 ( 8.5%)
   complete  1,935 (51.4%)  86,475.98 (82.5%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,829 (48.6%)  18,314.63 (17.5%)

     volume              1,003,648.55
  disbursed                985,333.92 (98.2)%
     target                983,575.58 (98.0)%
   distance                 -1,758.34
$
```

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 537 masspay
   6 twitter
   2 coinbase
   1 patreon
   1 github
$
```

**Comment by chadwhitacre** [comment]
I took a pass through accounts with balances >= $100 and paid out 10 of them in https://github.com/gratipay/logs/commit/0b117bf64a00f9ef323fc2bb469b530beb498f79. Remaining:

| n | status | notes |
|----:|:--------|:-------|
| 12 | pending | manually contacted (email, Reddit, etc.) |
| 3  | failed | can't find any more contact info |
| 1  | failed | suspicious |

```
$ ./tick.py 
                 n                $
             ------------- ------------------
      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    535 (14.2%)   6,996.30 ( 6.7%)
   complete  1,947 (51.7%)  88,359.22 (84.3%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,817 (48.3%)  16,431.39 (15.7%)

     volume              1,003,648.55
  disbursed                987,217.16 (98.4)%
     target                983,575.58 (98.0)%
   distance                 -3,641.58
$
```

https://github.com/gratipay/logs/commit/4a31f68a923bf9a126b6f6d03a41b31a7aedd7bd:

```
$ grep pending payouts.csv | cut -d, -f 2 | sort | uniq -c | sort -nr
 517 masspay
   6 email
   4 twitter
   2 slack
   2 patreon
   2 coinbase
   1 reddit
   1 github
$
```

**Comment by chadwhitacre** [comment]
https://twitter.com/chadwhitacre_/status/1081331795596066817

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/eb9caa3c7430fc84917abdf537e080b4c5729ee9

The coinbase ones are all but certain to clear, the patreon ones are close behind.

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                987,488.82 (98.4)%
     target                988,593.82 (98.5)%
   distance                  1,105.00

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    534 (14.2%)   6,724.64 ( 6.4%)
   complete  1,948 (51.8%)  88,630.88 (84.6%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,816 (48.2%)  16,159.73 (15.4%)

                 n                $
pending ---  ------------- ------------------

    masspay    516 (13.7%)   4,014.09 ( 3.8%)
      email      6 ( 0.2%)     908.32 ( 0.9%)
     github      1 ( 0.0%)     499.37 ( 0.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/016f4fc57ecbf589522e248ba87e767f119cabe9

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                987,887.26 (98.4)%
     target                988,593.82 (98.5)%
   distance                    706.56

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    532 (14.1%)   6,326.20 ( 6.0%)
   complete  1,950 (51.8%)  89,029.32 (85.0%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,814 (48.2%)  15,761.29 (15.0%)

                 n                $
pending ---  ------------- ------------------

    masspay    516 (13.7%)   4,014.09 ( 3.8%)
      email      4 ( 0.1%)     509.88 ( 0.5%)
     github      1 ( 0.0%)     499.37 ( 0.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
I heard from someone stating that `foo@example.com` was the wrong address to send to, and `foo+paypal@example.com` is the correct address. I've heard from one or two others indicating that folks have claimed their email address out from under them on PayPal. In those cases I was able to cancel the payment to the bad address before resending the good address. In this case, however, the payment to `foo@example.com` is complete and I can't refund it. Believing the person, I've gone ahead and resent the payment to the correct address. The name on the completed payment does not match the person's name, and PayPal does classify it as "Non-U.S.-non-verified." I've opened a dispute with PayPal to see if we can recover the bad payment.

**Comment by chadwhitacre** [comment]
Also PayPal web UI is hosed right now? 🤔 🙄

![screen shot 2019-01-07 at 8 40 04 pm](https://user-images.githubusercontent.com/134455/50804550-6e963480-12bc-11e9-921d-789cb72cc184.png)

![screen shot 2019-01-07 at 8 39 52 pm](https://user-images.githubusercontent.com/134455/50804551-6e963480-12bc-11e9-863e-93b3fef7db2f.png)

**Comment by chadwhitacre** [comment]
🤔

![screen shot 2019-01-07 at 8 43 15 pm](https://user-images.githubusercontent.com/134455/50804625-e1071480-12bc-11e9-90c0-bf5b627acb59.png)

**Comment by chadwhitacre** [comment]
Back!

https://www.paypal.com/activity/payment/2S370151T8621000W

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/e3ff0e22c987b6c50254aff53fd5484337913159

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                988,722.52 (98.5)%
     target                988,593.82 (98.5)%
   distance                   -128.70

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    527 (14.0%)   5,490.94 ( 5.2%)
   complete  1,955 (51.9%)  89,864.58 (85.8%)
     failed  1,282 (34.1%)   9,435.09 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,809 (48.1%)  14,926.03 (14.2%)

                 n                $
pending ---  ------------- ------------------

    masspay    513 (13.6%)   3,809.13 ( 3.6%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/6a35afc186496d72ddb9b2bd1517102372c0b40d

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                988,727.80 (98.5)%
     target                989,597.47 (98.6)%
   distance                    869.67

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    524 (13.9%)   5,449.16 ( 5.2%)
   complete  1,957 (52.0%)  89,869.86 (85.8%)
     failed  1,283 (34.1%)   9,471.59 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,807 (48.0%)  14,920.75 (14.2%)

                 n                $
pending ---  ------------- ------------------

    masspay    510 (13.5%)   3,767.35 ( 3.6%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/affc144bec151e5e903f3d50787cf20a68cc3598

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                988,862.01 (98.5)%
     target                989,597.47 (98.6)%
   distance                    735.46

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    513 (13.6%)   5,314.95 ( 5.1%)
   complete  1,968 (52.3%)  90,004.07 (85.9%)
     failed  1,283 (34.1%)   9,471.59 ( 9.0%)
      total  3,764         104,790.61
 incomplete  1,796 (47.7%)  14,786.54 (14.1%)

                 n                $
pending ---  ------------- ------------------

    masspay    499 (13.3%)   3,633.14 ( 3.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
Re: https://github.com/gratipay/inside.gratipay.com/issues/1205#issuecomment-452144117 ...

> After having reviewed all the details, and based on the information we have to date, we have to deny your case. This decision was made because your billing dispute doesn't match our qualification for a billing dispute.
>
> [...]
>
> If you have information that may help us decide this case in your favor, please send it to us.

I did.

**Comment by chadwhitacre** [comment]
Coinbase are throwing up roadblocks now, too:

> Your ability to trade and transact is disabled
>
> Your recent purchase of digital assets has failed. As a result, the ability to trade and transact is disabled. In 5 days, we will proactively initiate an automatic recovery.

I tried to add my personal debit card (PayPal credit card doesn't work) and they require verification amounts but there's only one in my account:

![screen shot 2019-01-10 at 9 05 38 pm](https://user-images.githubusercontent.com/134455/51008842-83bcce80-151b-11e9-8d9f-1d4c269e31cb.png)

Their [help](https://support.coinbase.com/customer/en/portal/articles/2739613-where-are-my-payment-method-verification-amounts-) says the 1.00 is to be ignored (I tried entering 00, believe me). Calling Citizens ...

Also kicked off a re-link of New Alliance. That'll take 2-3 days to see verification amounts, plus days to transfer I guess.

**Comment by chadwhitacre** [comment]
Citizens no help. 😞

**Comment by chadwhitacre** [comment]
Filed a support request with Coinbase. Followed up with a phone call but they say they don't support account-specific requests over the phone.

**Comment by chadwhitacre** [comment]
Coinbase unblocked, have both personal debit card and Gratipay's New Alliance bank account linked. 👍 I think I'm going to redo the transactions since the value has decreased over time.

PayPal re: dispute: "After having reviewed all the details, and based on the information we have to date, we have to deny your case. This decision was made because your billing dispute doesn't match our qualification for a billing dispute." 😞

First masspay completed; https://github.com/gratipay/logs/commit/d429cd5c10925442b939f4d2566ee561d0faf10f; I think I will wait for all to fail before doing another masspay.

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                988,921.20 (98.5)%
     target                989,597.47 (98.6)%
   distance                    676.27

                 n                $
big flush -  ------------- ------------------

      ready    146 ( 3.9%)   1,127.29 ( 1.1%)
    pending    108 ( 2.9%)   2,423.86 ( 2.3%)
   complete  1,971 (52.4%)  90,063.26 (85.9%)
     failed  1,539 (40.9%)  11,176.20 (10.7%)
      total  3,764         104,790.61
 incomplete  1,793 (47.6%)  14,727.35 (14.1%)

                 n                $
pending ---  ------------- ------------------

    masspay     94 ( 2.5%)     742.05 ( 0.7%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    146 ( 3.9%)
     target                  1,127.29
  remainder               -      0.31
                           ----------
     actual                  1,126.98 ( 1.1%)
        fee               -     21.43
                           ----------
        net                  1,105.55

$ 
```

**Comment by chadwhitacre** [comment]
Okay, I'm pretty sure this belongs here? I have received an anonymous Amazon package in the mail containing two books:

- [_Daughters of the Lake_](https://www.amazon.com/-/dp/1503900827) (a recent bestseller), and
- [_I am Jesus, whom you persecute_](https://www.amazon.com/-/dp/1794083219) (even more recent, self-published, and listed for $250!).

I have come to believe that this is a gift from someone who received a Gratipay payout, and that this person is the author of the self-published book. Gosh. Gosh!

Armstrong! THANK YOU!!!  🙇

**Comment by chadwhitacre** [comment]
Second masspay complete; https://github.com/gratipay/logs/commit/98fcd5a46876db3c18194c6811b8b1cc0573902e.

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                988,921.20 (98.5)%
     target                989,597.47 (98.6)%
   distance                    676.27

                 n                $
big flush -  ------------- ------------------

      ready    148 ( 3.9%)   1,245.55 ( 1.2%)
    pending    101 ( 2.7%)   2,177.87 ( 2.1%)
   complete  1,971 (52.4%)  90,063.26 (85.9%)
     failed  1,544 (41.0%)  11,303.93 (10.8%)
      total  3,764         104,790.61
 incomplete  1,793 (47.6%)  14,727.35 (14.1%)

                 n                $
pending ---  ------------- ------------------

    masspay     87 ( 2.3%)     496.06 ( 0.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
   coinbase      2 ( 0.1%)     240.12 ( 0.2%)
    patreon      2 ( 0.1%)     226.80 ( 0.2%)
     reddit      1 ( 0.0%)     103.25 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    148 ( 3.9%)
     target                  1,245.55
  remainder               -      0.31
                           ----------
     actual                  1,245.24 ( 1.2%)
        fee               -     23.39
                           ----------
        net                  1,221.85

$
```

**Comment by chadwhitacre** [comment]
Coinbase is a PITA. I've managed to complete one payout, three weeks after starting it and the price of BTC has been falling. 😖

**Comment by chadwhitacre** [comment]
Two more Coinbase payouts still hanging out there.

**Comment by chadwhitacre** [comment]
Coinbase: done! 💃

https://github.com/gratipay/logs/commit/8a7a174a3c8f45851fb456e9b99893e13d123382

Also resolved one more PayPal.

**Comment by chadwhitacre** [comment]
```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                989,468.27 (98.6)%
     target                989,597.47 (98.6)%
   distance                    129.20

                 n                $
big flush -  ------------- ------------------

      ready    147 ( 3.9%)   1,215.55 ( 1.2%)
    pending     97 ( 2.6%)   1,704.70 ( 1.6%)
   complete  1,978 (52.6%)  90,610.33 (86.5%)
     failed  1,542 (41.0%)  11,260.03 (10.7%)
      total  3,764         104,790.61
 incomplete  1,786 (47.4%)  14,180.28 (13.5%)

                 n                $
pending ---  ------------- ------------------

    masspay     87 ( 2.3%)     496.06 ( 0.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      4 ( 0.1%)     310.20 ( 0.3%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    147 ( 3.9%)
     target                  1,215.55
  remainder               -      0.31
                           ----------
     actual                  1,215.24 ( 1.2%)
        fee               -     22.80
                           ----------
        net                  1,192.44

$
```

**Comment by chadwhitacre** [comment]
lol :o)

> Your account has been identified as having $0.10 in your Dwolla balance as of January 23, 2019. Please take a moment to withdraw those funds to your bank account.

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/cc604325d6a84026fcbe94b5c0a7db6f40b99887

98.6% 💃

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                989,701.27 (98.6)%
     target                989,597.47 (98.6)%
   distance                   -103.80

                 n                $
big flush -  ------------- ------------------

      ready    147 ( 3.9%)   1,215.55 ( 1.2%)
    pending     94 ( 2.5%)   1,471.70 ( 1.4%)
   complete  1,981 (52.6%)  90,843.33 (86.7%)
     failed  1,542 (41.0%)  11,260.03 (10.7%)
      total  3,764         104,790.61
 incomplete  1,783 (47.4%)  13,947.28 (13.3%)

                 n                $
pending ---  ------------- ------------------

    masspay     87 ( 2.3%)     496.06 ( 0.5%)
      slack      2 ( 0.1%)     422.49 ( 0.4%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    147 ( 3.9%)
     target                  1,215.55
  remainder               -      0.31
                           ----------
     actual                  1,215.24 ( 1.2%)
        fee               -     22.80
                           ----------
        net                  1,192.44

$
```

**Comment by chadwhitacre** [comment]
> Okay, I'm pretty sure this belongs here?

Aaaaaaand this is fraud. I have an unexpected $277.76 charge on my card. Factor in media mail and that's the price of those two books:

![screen shot 2019-01-23 at 10 45 02 pm](https://user-images.githubusercontent.com/134455/51653218-8cbc8f80-1f60-11e9-8fd2-2cf7a3105e8a.png)

![screen shot 2019-01-23 at 10 38 27 pm](https://user-images.githubusercontent.com/134455/51653021-a8736600-1f5f-11e9-9b14-36a8361d1772.png)

I was traveling in Denver Jan 15-18. I actually did have my card stopped while I was out there, I figured it was a false positive due to traveling. I guess it was a true positive due to traveling! 😮

**Comment by chadwhitacre** [comment]
https://krebsonsecurity.com/2018/02/money-laundering-via-author-impersonation-on-amazon/

**Comment by chadwhitacre** [comment]
Paid out a $400+ balance!

Now only six > $100 left!

All are < $200!

Three are failed and the other three are likely to fail as well.

```
gratipay-bak=# select balance, status from tmp where status != 'complete' and balance > 100 order by balance desc;
┌─────────┬─────────┐
│ balance │ status  │
├─────────┼─────────┤
│  165.75 │ failed  │
│  162.33 │ failed  │
│  155.00 │ failed  │
│  147.00 │ pending │
│  118.00 │ pending │
│  113.95 │ pending │
└─────────┴─────────┘
(6 rows)
```

https://github.com/gratipay/logs/commit/cc604325d6a84026fcbe94b5c0a7db6f40b99887

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                990,123.76 (98.7)%
     target                991,604.77 (98.8)%
   distance                  1,481.01

                 n                $
big flush -  ------------- ------------------

      ready    147 ( 3.9%)   1,215.55 ( 1.2%)
    pending     92 ( 2.4%)   1,049.21 ( 1.0%)
   complete  1,983 (52.7%)  91,265.82 (87.1%)
     failed  1,542 (41.0%)  11,260.03 (10.7%)
      total  3,764         104,790.61
 incomplete  1,781 (47.3%)  13,524.79 (12.9%)

                 n                $
pending ---  ------------- ------------------

    masspay     87 ( 2.3%)     496.06 ( 0.5%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    147 ( 3.9%)
     target                  1,215.55
  remainder               -      0.31
                           ----------
     actual                  1,215.24 ( 1.2%)
        fee               -     22.80
                           ----------
        net                  1,192.44

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/8aa38e119b097cb7f96462f7dfe5596f56c9a1be

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                990,123.76 (98.7)%
     target                991,604.77 (98.8)%
   distance                  1,481.01

                 n                $
big flush -  ------------- ------------------

      ready    173 ( 4.6%)   1,346.41 ( 1.3%)
    pending      9 ( 0.2%)     663.49 ( 0.6%)
   complete  1,983 (52.7%)  91,265.82 (87.1%)
     failed  1,599 (42.5%)  11,514.89 (11.0%)
      total  3,764         104,790.61
 incomplete  1,781 (47.3%)  13,524.79 (12.9%)

                 n                $
pending ---  ------------- ------------------

      email      3 ( 0.1%)     378.95 ( 0.4%)
    masspay      4 ( 0.1%)     110.34 ( 0.1%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay    173 ( 4.6%)
     target                  1,346.41
  remainder               -      0.38
                           ----------
     actual                  1,346.03 ( 1.3%)
        fee               -     25.06
                           ----------
        net                  1,320.97

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/a20415ae412fbe695d0e51f7b11602e290fe999b

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                990,864.25 (98.7)%
     target                991,604.77 (98.8)%
   distance                    740.52

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending    107 ( 2.8%)   1,253.26 ( 1.2%)
   complete  2,055 (54.6%)  92,006.31 (87.8%)
     failed  1,602 (42.6%)  11,531.04 (11.0%)
      total  3,764         104,790.61
 incomplete  1,709 (45.4%)  12,784.30 (12.2%)

                 n                $
pending ---  ------------- ------------------

    masspay    102 ( 2.7%)     700.11 ( 0.7%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

$
```

**Comment by chadwhitacre** [comment]
This has been wildly successful. I am grateful for that.

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/b0a9f5b380eeec492b68363e6ab2b187f0afb6e2

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                990,956.03 (98.7)%
     target                991,604.77 (98.8)%
   distance                    648.74

                 n                $
big flush -  ------------- ------------------

      ready      1 ( 0.0%)       4.80 ( 0.0%)
    pending    101 ( 2.7%)   1,156.68 ( 1.1%)
   complete  2,060 (54.7%)  92,098.09 (87.9%)
     failed  1,602 (42.6%)  11,531.04 (11.0%)
      total  3,764         104,790.61
 incomplete  1,704 (45.3%)  12,692.52 (12.1%)

                 n                $
pending ---  ------------- ------------------

    masspay     96 ( 2.6%)     603.53 ( 0.6%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay      1 ( 0.0%)
     target                      4.80
  remainder               -      0.00
                           ----------
     actual                      4.80 ( 0.0%)
        fee               -      0.09
                           ----------
        net                      4.71

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/82ac4d9ab3bf6da73436d6253bb2d87207dddf37

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,011.35 (98.7)%
     target                991,604.77 (98.8)%
   distance                    593.42

                 n                $
big flush -  ------------- ------------------

      ready      1 ( 0.0%)       4.80 ( 0.0%)
    pending     99 ( 2.6%)   1,101.36 ( 1.1%)
   complete  2,062 (54.8%)  92,153.41 (87.9%)
     failed  1,602 (42.6%)  11,531.04 (11.0%)
      total  3,764         104,790.61
 incomplete  1,702 (45.2%)  12,637.20 (12.1%)

                 n                $
pending ---  ------------- ------------------

    masspay     94 ( 2.5%)     548.21 ( 0.5%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay      1 ( 0.0%)
     target                      4.80
  remainder               -      0.00
                           ----------
     actual                      4.80 ( 0.0%)
        fee               -      0.09
                           ----------
        net                      4.71

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/14d98e9e0141b21efbad78d3f63228c2271e0809

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,044.74 (98.7)%
     target                991,604.77 (98.8)%
   distance                    560.03

                 n                $
big flush -  ------------- ------------------

      ready      1 ( 0.0%)       4.80 ( 0.0%)
    pending     91 ( 2.4%)   1,067.97 ( 1.0%)
   complete  2,070 (55.0%)  92,186.80 (88.0%)
     failed  1,602 (42.6%)  11,531.04 (11.0%)
      total  3,764         104,790.61
 incomplete  1,694 (45.0%)  12,603.81 (12.0%)

                 n                $
pending ---  ------------- ------------------

    masspay     86 ( 2.3%)     514.82 ( 0.5%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay      1 ( 0.0%)
     target                      4.80
  remainder               -      0.00
                           ----------
     actual                      4.80 ( 0.0%)
        fee               -      0.09
                           ----------
        net                      4.71

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/2403c60633f92ca07c39b68147ced6965ff6bc24

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,052.86 (98.7)%
     target                991,604.77 (98.8)%
   distance                    551.91

                 n                $
big flush -  ------------- ------------------

      ready      1 ( 0.0%)       4.80 ( 0.0%)
    pending     89 ( 2.4%)   1,059.85 ( 1.0%)
   complete  2,072 (55.0%)  92,194.92 (88.0%)
     failed  1,602 (42.6%)  11,531.04 (11.0%)
      total  3,764         104,790.61
 incomplete  1,692 (45.0%)  12,595.69 (12.0%)

                 n                $
pending ---  ------------- ------------------

    masspay     84 ( 2.2%)     506.70 ( 0.5%)
      email      3 ( 0.1%)     378.95 ( 0.4%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay      1 ( 0.0%)
     target                      4.80
  remainder               -      0.00
                           ----------
     actual                      4.80 ( 0.0%)
        fee               -      0.09
                           ----------
        net                      4.71

$
```

**Comment by chadwhitacre** [comment]
Fourth MasspPay complete. Should we run a fifth? 🤔

https://github.com/gratipay/logs/commit/5a22ac902fa1ffeac96602a31f3de1cd47985c6c

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,053.02 (98.7)%
     target                991,604.77 (98.8)%
   distance                    551.75

                 n                $
big flush -  ------------- ------------------

      ready     22 ( 0.6%)     120.01 ( 0.1%)
    pending     10 ( 0.3%)     663.56 ( 0.6%)
   complete  2,073 (55.1%)  92,195.08 (88.0%)
     failed  1,659 (44.1%)  11,811.96 (11.3%)
      total  3,764         104,790.61
 incomplete  1,691 (44.9%)  12,595.53 (12.0%)

                 n                $
pending ---  ------------- ------------------

      email      3 ( 0.1%)     378.95 ( 0.4%)
    masspay      5 ( 0.1%)     110.41 ( 0.1%)
    patreon      1 ( 0.0%)      97.00 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay     22 ( 0.6%)
     target                    120.01
  remainder               -      0.06
                           ----------
     actual                    119.95 ( 0.1%)
        fee               -      2.36
                           ----------
        net                    117.59

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/86975b1e74c022e353f1819d68fd31da7c701eb1

I actually double-paid this $97 because the person released two things on Patreon and I didn't pay close enough attention. 😞

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,150.02 (98.8)%
     target                991,604.77 (98.8)%
   distance                    454.75

                 n                $
big flush -  ------------- ------------------

      ready     22 ( 0.6%)     120.01 ( 0.1%)
    pending      9 ( 0.2%)     566.56 ( 0.5%)
   complete  2,074 (55.1%)  92,292.08 (88.1%)
     failed  1,659 (44.1%)  11,811.96 (11.3%)
      total  3,764         104,790.61
 incomplete  1,690 (44.9%)  12,498.53 (11.9%)

                 n                $
pending ---  ------------- ------------------

      email      3 ( 0.1%)     378.95 ( 0.4%)
    masspay      5 ( 0.1%)     110.41 ( 0.1%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)

                 n                $
masspay ---  ------------- ------------------

    masspay     22 ( 0.6%)
     target                    120.01
  remainder               -      0.06
                           ----------
     actual                    119.95 ( 0.1%)
        fee               -      2.36
                           ----------
        net                    117.59

$
```

**Comment by chadwhitacre** [comment]
> Aaaaaaand somehow I double-paid you (I think?), even though it's not April 1 yet. 😞
> 
> I intended to pay you $97. Can you see if I paid you twice and refund the second payment?
> 
> Thanks and sorry for the trouble.

https://www.patreon.com/messages/3804593

**Comment by chadwhitacre** [comment]
> Should we run a fifth? 🤔

[Yes!](https://github.com/gratipay/inside.gratipay.com/issues/1219#issuecomment-472674134) 💃

https://github.com/gratipay/logs/commit/af4c02fd45253b8fc2cc8b1055adc520523be968

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,181.79 (98.8)%
     target                991,604.77 (98.8)%
   distance                    422.98

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending     24 ( 0.6%)     501.44 ( 0.5%)
   complete  2,078 (55.2%)  92,323.85 (88.1%)
     failed  1,662 (44.2%)  11,965.32 (11.4%)
      total  3,764         104,790.61
 incomplete  1,686 (44.8%)  12,466.76 (11.9%)

                 n                $
pending ---  ------------- ------------------

      email      3 ( 0.1%)     378.95 ( 0.4%)
    twitter      1 ( 0.0%)      77.20 ( 0.1%)
    masspay     20 ( 0.5%)      45.29 ( 0.0%)

$
```

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/4161aa3c871f4d2a80d08f8c2d6e78f19b6d508a

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,181.79 (98.8)%
     target                991,604.77 (98.8)%
   distance                    422.98

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending     17 ( 0.5%)      39.49 ( 0.0%)
   complete  2,078 (55.2%)  92,323.85 (88.1%)
     failed  1,669 (44.3%)  12,427.27 (11.9%)
      total  3,764         104,790.61
 incomplete  1,686 (44.8%)  12,466.76 (11.9%)

                 n                $
pending ---  ------------- ------------------

    masspay     17 ( 0.5%)      39.49 ( 0.0%)

$
```

**Comment by chadwhitacre** [comment]
> Patreon refunded $97.00 USD from your purchase on March 13, 2019.
> 
> The money has been refunded to your PayPal balance.

👍

**Comment by chadwhitacre** [comment]
https://github.com/gratipay/logs/commit/9f83d60085d3dc35c6ef29847355cda1f45a1049

I think there is going to be one failure with one more email address to try, for < $10. I think I may skip or try manually. Almost done, just another month! 😬

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,182.61 (98.8)%
     target                991,604.77 (98.8)%
   distance                    422.16

                 n                $
big flush -  ------------- ------------------

      ready      0 ( 0.0%)       0.00 ( 0.0%)
    pending     16 ( 0.4%)      38.67 ( 0.0%)
   complete  2,079 (55.2%)  92,324.67 (88.1%)
     failed  1,669 (44.3%)  12,427.27 (11.9%)
      total  3,764         104,790.61
 incomplete  1,685 (44.8%)  12,465.94 (11.9%)

                 n                $
pending ---  ------------- ------------------

    masspay     16 ( 0.4%)      38.67 ( 0.0%)

$
```

**Comment by chadwhitacre** [comment]
Fifth MassPay completed. Not worth doing a sixth MassPay, we are down to two payouts, one for $1.34 and the other for 39¢. We are done with the big flush! Now to donate the rest ...

https://github.com/gratipay/logs/commit/f9284b9043666c551cd53582767d167fff73f639

```
$ ./tick.py 

                 n                $
overall ---  ------------- ------------------

     volume              1,003,648.55
  disbursed                991,182.61 (98.8)%
     target                991,604.77 (98.8)%
   distance                    422.16

                 n                $
big flush -  ------------- ------------------

      ready      2 ( 0.1%)       1.77 ( 0.0%)
    pending      0 ( 0.0%)       0.00 ( 0.0%)
   complete  2,079 (55.2%)  92,324.67 (88.1%)
     failed  1,683 (44.7%)  12,464.17 (11.9%)
      total  3,764         104,790.61
 incomplete  1,685 (44.8%)  12,465.94 (11.9%)

                 n                $
masspay ---  ------------- ------------------

    masspay      2 ( 0.1%)
     target                      1.77
  remainder               -      0.00
                           ----------
     actual                      1.77 ( 0.0%)
        fee               -      0.04
                           ----------
        net                      1.73

$
```

**Comment by chadwhitacre** [comment]
Regained access to New Alliance. Balance remaining is $13,223.95.

**Comment by chadwhitacre** [comment]
That's $758.01 more than what is supposedly remaining post-payouts. ¯\\\_(ツ)\_/¯

**Comment by chadwhitacre** [comment]
Here is what I propose to donate:

| Recipient | Amount ($) |
|:-------|-------:|
| [Liberapay](https://liberapay.com/) | 3,306.00 |
| [OSI](https://opensource.org/) | 3,306.00 |
| [FSF](https://www.fsf.org/) | 3,306.00 |
| [Abstractions](https://abstractions.io/) | 3,305.95 |

**Comment by chadwhitacre** [comment]
There's another $4,080.05 in PayPal! 😲

That's $17,304.00 total, or $4,326 each when split four ways.

I don't have a debit card for the New Alliance account, Liberapay, OSI, and FSF all accept PayPal. Perhaps Abstractions can, too? 🤔

**Comment by chadwhitacre** [comment]
Contacted Abstractions, and kicked off a transfer of all remaining funds to PayPal. Once that clears in a few days I will make the payouts!

![transfer](https://user-images.githubusercontent.com/134455/63040364-4f373980-be93-11e9-9a1b-8089831f34e8.png)

**Comment by chadwhitacre** [comment]
Current proposal:

| Recipient | Amount ($) |
|:-------|-------:|
| [Liberapay](https://liberapay.com/) | 4,326.00 |
| [OSI](https://opensource.org/) | 4,326.00 |
| [FSF](https://www.fsf.org/) | 4,326.00 |
| [Abstractions](https://abstractions.io/) | 4,326.00 |

**Comment by chadwhitacre** [comment]
I didn't make it to New Alliance today and I'm not sure when I'll be able to. I ordered a new card from PayPal instead. When I tried using it before I wasn't able to in some cases. Also there is a $3,000 per day spending limit. So that'll be two transactions ... if DonorBox accepts PayPal Debit! 🙈

**Comment by chadwhitacre** [comment]
Money has cleared PayPal, balance is $17,304. Activated card, tried it on https://donorbox.org/codeandsupplyfund-abstractions2. Card declined. 😞

Using my personal credit card, will reimburse.

**Comment by chadwhitacre** [comment]
> $3,000 per day spending limit

Oh right! Probs why declined. 😬

**Comment by chadwhitacre** [comment]
![Screen Shot 2019-08-23 at 7 10 34 AM](https://user-images.githubusercontent.com/134455/63588695-22f77900-c575-11e9-8074-fb72ebbb42a0.png)

**Comment by chadwhitacre** [comment]
Reimbursed myself.

**Comment by chadwhitacre** [comment]
One down, three to go. I wanted to get Abstractions done today because it is the last day of the conference. I will try to follow up with the other three this weekend.

**Comment by chadwhitacre** [comment]
Donation to Liberapay complete.

**Comment by chadwhitacre** [comment]
Donation to OSI complete.

**Comment by chadwhitacre** [comment]
![Screen Shot 2019-08-26 at 8 53 42 PM](https://user-images.githubusercontent.com/134455/63732742-a6e68500-c843-11e9-9528-50b0be1de06a.png)

**Comment by chadwhitacre** [comment]
P.S. Donation to FSF also complete. 😬

!m * 💃
